src/components/ActionCard.tsx

```diff
@@ -62,16 +62,27 @@ const Card = styled(Link)`
   }
 `
 
-const ActionCard = ({
-  to,
-  alt,
-  image,
-  title,
-  description,
-  children,
-  className,
-  isRight,
-  isBottom = true,
+export interface IProps {
```

Comment:
You made everything optional but there are some props that we do want to make them as required.
How about this?
```ts
export interface IProps {
  to: string
  alt?: string
  image: string
  title: string
  description: string
  className?: string
  isRight?: boolean
  isBottom?: boolean
}
```